libs/designer-ui/src/lib/dynamicallyaddedparameter/helper.ts

```diff
@@ -190,12 +230,26 @@ export function serialize(props: DynamicallyAddedParameterProps[]): ValueSegment
   ];
 }
 
-export function getEmptySchemaValueSegmentForInitialization(useStaticInputs: boolean) {
-  let rootObject: { type: string; properties: unknown; required: string[] } = {
-    type: 'object',
-    properties: {},
-    required: [],
-  };
+export function getEmptySchemaValueSegmentForInitialization(useStaticInputs: boolean, isManualTrigger: boolean) {
+  let rootObject;
+  if (isManualTrigger) {
```

Comment:
other way around?

Reply:
Same comment as above, defaulting now to false so now it's fixed